File: .environment/database/etl/Test_CAN_DB.cs
using System;
using System.Data.SqlClient;

class Test_CAN_DB
{
    static void Main()
    {
        // Cadena de conexión a la base de datos
        string connectionString = "Server=PAT-PC;Database=CAN_DB;User Id=pat_mic;Password=michael;";

        // Establecer la conexión
        try
        {
            using (SqlConnection connection = new SqlConnection(connectionString))
            {
                connection.Open();

                // Consulta SQL para seleccionar datos de la tabla Homologacion
                string query = "SELECT * FROM Homologacion";

                // Ejecutar la consulta
                using (SqlCommand command = new SqlCommand(query, connection))
                {
                    using (SqlDataReader reader = command.ExecuteReader())
                    {
                        // Obtener los resultados
                        while (reader.Read())
                        {
                            // Imprimir los resultados
                            for (int i = 0; i < reader.FieldCount; i++)
                            {
                                Console.Write(reader[i] + " ");
                            }
                            Console.WriteLine();
                        }
                    }
                }
            }
        }
        catch (SqlException e)
        {
            Console.WriteLine("Error al conectar a la base de datos: " + e.Message);
        }
    }
}

File: .environment/database/etl/test_CAN_DB.py
import pyodbc
# Cadena de conexión a la base de datos
driver  = 'SQL Server'
server  = 'PAT-PC'
db1     = 'CAN_DB'
tcon    = 'yes'
uname   = 'pat_mic'
pword   = 'michael'

# Establecer la conexión
try:
    conn = pyodbc.connect(driver='{SQL Server}', host=server, database=db1, trusted_connection=tcon, user=uname, password=pword)
    cursor = conn.cursor()

    # Consulta SQL para seleccionar datos de la tabla Homologacion
    query = "SELECT * FROM Homologacion"

    # Ejecutar la consulta
    cursor.execute(query)

    # Obtener los resultados
    rows = cursor.fetchall()

    # Imprimir los resultados
    for row in rows:
        print(row)

    # Cerrar la conexión
    conn.close()

except pyodbc.Error as e:
    print("Error al conectar a la base de datos:", e)
